server/storage/players.lua

```diff
@@ -369,6 +372,24 @@ RegisterCommand('convertjobs', function(source)
     TriggerEvent('qbx_core:server:jobsconverted')
 end, true)
 
+---Removes invalid groups from the player_groups table.
+RegisterCommand('cleanplayergroups', function(source)
```

Comment:
Also needs to handle checking for a valid grade as well.

Reply:
Also consider adding an opt-in convar to automatically run this on server start. Just need to make sure the jobs/gangs are loaded first.

Reply:
Fixed and convar added.